CHANGELOG.md

```diff
@@ -1,5 +1,20 @@
 # Changelog
 
+## 0.12.0
+
+0.12.0 adds support for running sidecar containers in the M3DB pods.
```

Comment:
Can you add something like "and makes clusters created by the operator use 1.0 M3DB configs by default"?